client/client_test.go

```diff
@@ -130,6 +130,24 @@ func TestDaemonUp(t *testing.T) {
 	assert.Equal(t, actualCommand, session.Calls[0])
 }
 
+func TestDaemonDown(t *testing.T) {
+	session := &mockSSHRunner{}
+	client := newMockSSHClient(session)
+	client.version = "latest"
+	client.IP = "0.0.0.0"
+	client.Daemon.Port = "4303"
+	script, err := ioutil.ReadFile("scripts/daemon-up.sh")
+	assert.Nil(t, err)
+	actualCommand := fmt.Sprintf(string(script), "latest", "4303", "0.0.0.0")
+
+	// Make sure the right command is run.
+	err = client.DaemonUp("latest")
+	err = client.DaemonDown()
+	assert.Nil(t, err)
+	println(actualCommand)
+	assert.Equal(t, actualCommand, session.Calls[0])
```

Comment:
wouldn't this actually be testing `daemon-up.sh`? see [line 139](https://github.com/ubclaunchpad/inertia/pull/573/files#diff-fce84029d346504e44323c30d21a3808R139) 🤔